apis/management.cattle.io/v3/rke_types.go

```diff
@@ -183,6 +183,8 @@ type ETCDService struct {
 	Retention string `yaml:"retention" json:"retention,omitempty" norman:"default=72h"`
 	// Etcd snapshot Creation period
 	Creation string `yaml:"creation" json:"creation,omitempty" norman:"default=12h"`
```

Comment:
It thought we were going to fix this and Retention to be an integer number of minutes?  Is that a future release?

Don't like string times with complicated parsing and validation.

Reply:
We will move this to a future iteration. For now, UI can set this to a number of hours.

Reply:
update: we are introducing retention and intervalHours parameters to the BackupConfig as per conversation with @vincent99 : https://github.com/rancher/types/pull/633/commits/487bca05840ccf16008babd4ade145541e6ce803#diff-623d9a24d8befd392620d1315c53f7d2R18

Reply:
@guangbochen We wanted to introduce a new format for these 2 parameters (hours for creation, number of snapshots for retention). But in order to not break api compatibility with the previous versions, we couldn't change the existing params, so we've introduced new ones in backupConfig, and those params would be applied to S3 backups (for now) only. 

> If both ETCDService and BackupConfig has Retention set with 12h and 6(num of backups), which one it should follow when removing the local backups

Old parameters should apply to the local backups only; new params from backupConfig should apply to the backups on S3.

Let me know if everything is clear @guangbochen 

 @moelsayed please confirm

Reply:
sure thanks @alena1108, I will add the s3 `backupConfig` to the rke-tools.

Reply:
@guangbochen we don't need to add it to the rke tools; we just have to change rke side code to parse the correct retention and creation parameters.